pyproject.toml

```diff
@@ -25,11 +25,7 @@ classifiers = [
   "Typing :: Typed",
 ]
 requires-python = ">= 3.11, < 4"
-dependencies = [
-  "googleapis-common-protos >= 1.56.4, < 2",
-  "googleapis-common-protos-stubs == 2.2.0",
-  "grpcio >= 1.51.1, < 2",
-]
+dependencies = ["grpcio >= 1.51.1, < 2", "protobuf >= 4.25.3, < 5"]
```

Comment:
Because we still use many `google.protobuf.xxx` messages that are included in the `protoc` compiler but still need the python part of it, before that dependency was indirectly pulled by `googleapis-common-protos`.